docs/examples/water_rocket/water_rocket.ipynb

```diff
@@ -0,0 +1,980 @@
+{
+ "cells": [
+  {
```

Comment:
Water Rocket is missing the section `Accessing the results`, which is the very last set of code blocks before the references.

Also, the N2 diagrams in this file aren't loading for me.

Reply:
Fixed